engine/runner/src/main/java/org/enso/runner/Main.java

```diff
@@ -628,6 +628,7 @@ private void createNew(
             authors,
             nil(),
             "",
+            Option$.MODULE$.empty(),
```

Comment:
This is the only change in `engine/runner`. Is that sufficient to switch to JVM mode when there is a project with `jvm: true` and one does:
```
enso --run the_project
```
? I am afraid that it is not sufficient.